### PR TITLE
Whitesource

### PR DESCRIPTION
**Pull request by andreaTP**
### What changes were proposed in this pull request?
Add the whitesource plugin

### Why are the changes needed?
Report licenses issues

### Does this PR introduce any user-facing change?
No

### How was this patch tested?
Locally will test on GH Actions with this PR

### CODE REVIEW

**Review by RayRoestenburg — APPROVED** [review]

File: core/project/WhitesourceLicensePlugin.scala [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
+  override def trigger = allRequirements
 
-object LicensePlugin extends AutoPlugin {
-  override def requires = plugins.JvmPlugin
-  override def trigger  = allRequirements
+  override lazy val projectSettings = Seq(
+    // do not change the value of whitesourceProduct
+    whitesourceProduct := "cloudflow",
+    whitesourceAggregateProjectName := {
+      "cloudflow-2.0-" + (
```
Comment (ennru) [inline core/project/WhitesourceLicensePlugin.scala:22]:
```suggestion
      "cloudflow-" + (
```
The version should come from `version` (as it does for non-snapshots below) and not be part of the `master` and `adhoc` updates.

**Review by ennru — COMMENTED** [review]
The GitHub workflow is not in this PR yet.

File: core/project/WhitesourceLicensePlugin.scala (hunk 1 above)
Comment (andreaTP) [reply core/project/WhitesourceLicensePlugin.scala]: updated! Need to wait for next RC release before merging (already triggered)

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**